honeybee_schema/geometry.py

```diff
@@ -0,0 +1,58 @@
+"""Geometry objects for model."""
+from pydantic import Field, constr, conlist
+from typing import List
+from ._base import NoExtraBaseModel
+
+
+class Plane(NoExtraBaseModel):
+
+    type: constr(regex='^Plane$') = 'Plane'
+
+    n: List[float] = Field(
+        ...,
+        description="Plane normal as 3 (x, y, z) values.",
+        min_items=3,
+        max_items=3
+    )
+
+    o: List[float] = Field(
+        ...,
+        description="Plane origin as 3 (x, y, z) values",
+        min_items=3,
+        max_items=3
+    )
+
+    x: List[float] = Field(
+        default=None,
+        description="Plane x-axis as 3 (x, y, z) values. If None, it is autocalculated.",
+        min_items=3,
+        max_items=3
+    )
+
+
+class Face3D(NoExtraBaseModel):
```

Comment:
Are these the only two geometry object that we have in the schema? If there are more objects then let's make sure to move all of them here. Otherwise disregard this comment.

Reply:
They are the only objects right now, @mostaphaRoudsari .  In the future, we'll add the sphere, cone and cylinder but there's no support for these currently in the Model schema. Everything is a Face3D.